### ISSUE TITLE
Add "Downloading Metadata" Status in QBT

### ISSUE THREAD

**Issue by drowse7en** [jerrymakesjelly/autoremove-torrents#99]
Sometimes, QBT stalls in the status, "downloading metadata" and never exits.

Please add support for this status as well.

**Comment by dantebarba** [comment]
+1!

**Comment by drowse7en** [comment]
@dantebarba Yes, I'm able to manage stalled torrents in "stalled" status, but not stalled torrents in "downloading metadata" status.

**Comment by wattseat** [comment]
> @dantebarba Yes, I'm able to manage stalled torrents in "stalled" status, but not stalled torrents in "downloading metadata" status.

Hey there. Was looking for a solution as well. I tried a few things and actually solved it. If you set status as “Unknown” it will pick those torrents stuck on “metadata”

Hope this helps.

**Comment by drowse7en** [comment]
> Hey there. Was looking for a solution as well. I tried a few things and actually solved it. If you set status as “Unknown” it will pick those torrents stuck on “metadata”

Thanks for the advice.  I'll definitely give it a try.

**Comment by ajlunis** [comment]
Did this ever work out for you guys?  I've tried adding the "Unknown" status and unfortunately it still won't remove torrents stuck in "Downloading metadata"

**Comment by wattseat** [comment]
> Did this ever work out for you guys? I've tried adding the "Unknown" status and unfortunately it still won't remove torrents stuck in "Downloading metadata"

Yes. Mine is working correctly with the “Unknown” status. Maybe your strategy isn’t picking up those torrents for other reason. For instance, setting last activity rule won’t work since those torrents don’t register any activity at all. 

Here is the config I use for those torrents. It’s not ideal, but it is what I could manage. Other combinations didn’t work for me because of the peculiarities of torrents with that status.

status:
- Unknown
- StalledDownload
remove: create_time > 86400 and progress < 0.01

**Comment by drowse7en** [comment]
> Did this ever work out for you guys? I've tried adding the "Unknown" status and unfortunately it still won't remove torrents stuck in "Downloading metadata"

Yes, this works for me using:

`task_remove_stalled:
  client: qbittorrent
  host: http://127.0.0.1:*****
  username: **********
  password: **********
  strategies:
    remove_stalled:
      categories:
        - tv-sonarr
        - movies-radarr
        - music-lidarr
        - books-readarr
      status:
        - StalledDownload
        - StalledUpload
        - Downloading
        - Uploading
        - Unknown
      remove: progress < 1
  delete_data: true`
